src/index.ts

```diff
@@ -1,164 +1,616 @@
+
+    const manners = options.manners || defaultMannersComponent(config)
+    const crypto = options.crypto || await defaultCryptoComponent(config.namespace)
+    const storage = options.storage || defaultStorageComponent(config.namespace)
+    const reference = options.reference || await defaultReferenceComponent({ crypto, manners, storage })
+
+    if (disableWnfs) {
+      if (staging) return FissionAuthBaseStaging.implementation({ crypto, reference, storage })
+      return FissionAuthBaseProduction.implementation({ crypto, reference, storage })
     } else {
-      const fs = options.loadFileSystem === false ?
-        undefined :
-        await loadFileSystem(rootPermissions, authedUsername)
-
-      return appState.scenarioAuthed(
-        authedUsername,
-        fs
-      )
+      if (staging) return FissionAuthWnfsStaging.implementation({ crypto, reference, storage })
+      return FissionAuthWnfsProduction.implementation({ crypto, reference, storage })
     }
+  }
+}
 
-  } else if (authedUsername) {
-    return appState.scenarioAuthed(
-      authedUsername,
-      undefined
-    )
-
-  } else {
-    return appState.scenarioNotAuthed()
+/**
+ * Predefined confidences configurations.
+ *
+ * If you want partial read and/or write access to the filesystem you'll want
+ * a "confidences" component. This component is responsible for requesting
+ * and receiving UCANs, read keys and namefilters from other sources to enable this.
+ *
+ * NOTE: This uses all the default components as the dependencies for the confidences component.
+ *       If you're, for example, using a non-default crypto component, you'll want to
+ *       pass that in here as a parameter as well.
+ *
+ *       Dependents: crypto, depot.
+ */
+export const confidences = {
+  /**
+   * A secure enclave in the form of a webnative app which serves as the root authority.
+   * Your app is redirect to the lobby where the user can create an account or link a device,
+   * and then request permissions to the user for reading or write to specific parts of the filesystem.
+   */
+  async fissionLobby(
+    config: Configuration,
+    options: {
+      staging?: boolean
+
+      // Dependents
+      crypto?: Crypto.Implementation
+      depot?: Depot.Implementation
+    } = {}
+  ): Promise<ConfidencesImpl.Implementation> {
+    const { staging } = options
+
+    const crypto = options.crypto || await defaultCryptoComponent(config.namespace)
+    const depot = options.depot || await defaultDepotComponent(config.namespace)
+
+    if (staging) return FissionLobbyStaging.implementation({ crypto, depot })
+    return FissionLobbyProduction.implementation({ crypto, depot })
+  }
+}
 
+/**
+ * Predefined depot configurations.
+ *
+ * The depot component gets data in and out your program.
+ * For example, say I want to load and then update a file system.
+ * The depot will get that file system data for me,
+ * and when updating it, bring the data to where it needs to be.
```

Comment:
```suggestion
 * and after updating it, send the data to where it needs to be.
```

Not at all sure about these ones. If I recall correctly, we tell the server "come and get the data". So perhaps this is more like

> and after updating it, will send the data or notify an external service to fetch the data.

Service also maybe isn't right? Consider it an initial idea. 😅

Reply:
Well, it's not the depot that notifies the "external service". My wording is probably confusing here. It's really the data root we update via the reference component that notifies the server to get the data from our app's depot.